third_party/terraform/utils/compute_instance_helpers.go

```diff
@@ -347,3 +348,31 @@ func flattenEnableDisplay(displayDevice *computeBeta.DisplayDevice) interface{}
 
 	return displayDevice.EnableDisplay
 }
+
+// Terraform doesn't correctly calculate changes on schema.Set, so we do it manually
+// https://github.com/hashicorp/terraform-plugin-sdk/issues/98
+func schedulingHasChange(d *schema.ResourceData) bool {
```

Comment:
Can you add a comment somewhere in the schema for scheduling that if changes get made, to update this too?